tests/test_utils/test_version_utils.py

```diff
@@ -4,19 +4,19 @@
 
 
 def test_digit_version():
-    assert digit_version('0.2.16') == (0, 2, 16)
-    assert digit_version('1.2.3') == (1, 2, 3)
-    assert digit_version('1.2.3rc0') == (1, 2, 2, 0)
-    assert digit_version('1.2.3rc1') == (1, 2, 2, 1)
-    assert digit_version('1.0rc0') == (1, -1, 0)
+    assert digit_version('0.2.16') == (0, 2, 16, 0, 0, 0)
+    assert digit_version('1.2.3') == (1, 2, 3, 0, 0, 0)
+    assert digit_version('1.2.3rc0') == (1, 2, 3, 0, -1, 0)
+    assert digit_version('1.2.3rc1') == (1, 2, 3, 0, -1, 1)
+    assert digit_version('1.0rc0') == (1, 0, 0, 0, -1, 0)
 
```

Comment:
Adding more test case would be great, like `1.5.0+cuda90_cudnn7.6.3_lms` you mentioned earlier

Reply:
Also test the comparison. Like rc0 < rc1, alpha < beta, etc.